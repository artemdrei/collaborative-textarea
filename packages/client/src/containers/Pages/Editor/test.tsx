import React from 'react';

import { shallow, ShallowWrapper } from 'enzyme';
import toJson from 'enzyme-to-json';

import Page from '.';

jest.mock('react-router-dom', () => ({
  withRouter: jest.fn(),
  useHistory: () => ({
    push: jest.fn(),
  }),
}));

describe('Editor page', () => {
  let wrapper: ShallowWrapper<null>;

  beforeEach(() => {
    wrapper = shallow(<Page />);
  });

  it('should match the snapshot', () => expect(toJson(wrapper)).toMatchSnapshot());
});
