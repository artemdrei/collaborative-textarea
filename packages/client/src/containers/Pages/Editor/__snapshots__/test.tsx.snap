// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Editor page should match the snapshot 1`] = `
<div>
  <Header />
  <div>
    <textarea
      id="#textarea"
      placeholder="Write message..."
    />
  </div>
</div>
`;
